share/gpodder/extensions/subscription_stats.py

```diff
@@ -40,46 +36,42 @@ def on_create_menu(self):
     def add_page(self, notebook, category, channels):
         scrolled = Gtk.ScrolledWindow()
         scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+        
 
         store = Gtk.ListStore(str, float, str, str, int)
         for average, name, edate, paused in channels:
-            average_flot = float(average)
             last = strftime('%x', localtime(edate))
             store.append([
                 ('%.1f' % round(average, 1)) if average > 0 else '?',
-                average_flot, ('‖ ' if paused else '') + name, last, edate,
+                average, ('‖ ' if paused else '') + name, last, edate,
             ])
 
         tree = Gtk.TreeView(model=store)
         scrolled.add(tree)
 
         dayscell = Gtk.CellRendererText()
-        dayscell.set_property('xalign', 1)
-        dayscell.set_property('alignment', Pango.Alignment.RIGHT)
```

Comment:
I can't find the conversation on why the days column was changed from right aligned to centered, but it is hard to compare numbers when centered. It works when all of the numbers have the same number of digits, but the weekly and yearly categories can have varying widths. Right aligning also pulls the numbers closer to the podcast name, making it easier to associate them.

Reply:
The alignment centered was tests from my part to see if resolved the issues with the Updated column (when was the last column), and was left in the last version, nevertheless I've changed also the position of the columns so next to the Days column are now the Updated column.
So it's enough to change again to dayscell.set_property('xalign', 1)
I also remove the dayscell.set_property('alignment', Pango.Alignment.RIGHT) because I think is not necessary, but I could be wrong :)

Reply:
You are correct, the alignment property is not needed here. It is only required if the cell can have more than one line.